doc/userdoc/contribute/styleguide/styleguide.rst

```diff
@@ -0,0 +1,654 @@
+
+* Avoid using "so" in sentences.
+
+* Try to keep sentences short, or break up long sentences with short ones.
+
+* Avoid lengthy paragraphs with more than 5 or 6 sentences.
+  If writing multiple paragraphs, they should be broken up by example code, figures, or bullet lists.
+
+* Keep in mind that texts should be skimmable.
+
+Pronouns
+~~~~~~~~
+
+* Use the pronouns "you" to indicate the reader and "we" to indicate NEST and its members.
+
+* Avoid the pronoun "I."
+
+* Avoid gendered terms (e.g., use "police officer" instead of "policeman").
+
+* Instead of "guys" or "girls" use inclusive language such as everyone, all,
+  members, or folks.
+
+* "They" is an acceptable singular third person pronoun
+  (see `the dictionary definition here <https://www.merriam-webster.com/dictionary/they>`_).
+
+How to write specific content in NEST
+-------------------------------------
+
+Numbers
+~~~~~~~
+
+* Numbers 0–9 should be spelled out, unless they are measurements or coordinates.
+
+* Numbers should be spelled out if they begin a sentence. In most cases, however,
+  the numeral/ordinal format is preferred.
+
+* We use the period for the decimal point (`57.45`).
+
+* The thousand separator is the comma except when showing a code example.
+
+   Example:
+
+   We have over 5,000 connections.
+   The number of connections is ``x = 5001``.
+
+* Make sure you use the correct unit (e.g., millivolts for voltage) and the
+  unit's symbol (mV).
+
+* For additional mathematical notation, use the :ref:`math role or directive <math_style>`.
+
+Lists
+~~~~~
+
+* Use the serial comma in lists.
+
+* Use numbered lists for step-by-step instructions only. Ensure that each step contains only one or two actions.
+
+* Use bullet lists if the number of items is extensive or each item is a long phrase or sentence.
+
+* If the text of a list forms a complete sentence, use proper punctuation and
+  end with period.
+
+* If the text of a list forms an incomplete sentence, do not end with period.
+
+* If the entire bullet/numbered list belongs to a sentence, end each item with a comma and the second-last item with ", and".
+
+* If the last item is the end of the sentence, end it with a period. Otherwise, use the punctuation required to correctly connect
+  to the remainder of the sentence.
+
+
+Abbreviations and acronyms
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Spell out acronyms on first appearance on each page or article it appears, for example: Random number generator (RNG).
+
+* If the abbreviation/acronym is well known (e.g., HTML), you do not need to spell
+  it out.
+
+* Use "e.g.," and "i.e.," only in parentheses and figure and table captions; otherwise, use "for example," and "that is,".
+  Note the use of comma following the terms.
+
+
+Contractions
+~~~~~~~~~~~~
+
+* Avoid contractions. For example, use "do not" instead of "don't".
+
+Commas
+~~~~~~
+
+* Use the serial comma (apples, bananas, and grapes) for lists.
+
+* Use the comma as separator for thousands (37,000).
+
```

Comment:
This is a duplicate of the information given in "Numbers" section. Maybe add a link there, because it holds more information.